src/mmw/js/src/modeling/gwlfe/subbasin/views.js

```diff
@@ -51,6 +58,52 @@ var ResultView = Marionette.LayoutView.extend({
             }),
         ]);
     },
+
+    getLayerSelector: function() {
+        return null;
+    }
+});
+
+var LayerSelectorView = Marionette.ItemView.extend({
+    // model: SubbasinDetailModel
```

Comment:
Just double-checking that this is a comment and not commented out code.

Reply:
Yes, following the pattern used elsewhere:

```
> ag -Q '// model:'
src/mmw/js/src/core/sidebarToggleControl.js
22:    // model: MapModel,

src/mmw/js/src/data_catalog/views.js
276:    // model: FormModel
620:    // model: ExpandableListModel
1152:    // model: PopoverControllerModel
1240:    // model: DateFilterModel
1279:    // model: CheckboxFilterModel

src/mmw/js/src/modeling/gwlfe/subbasin/views.js
17:    // model: ResultModel
68:    // model: SubbasinDetailModel
110:    // model: SubbasinTabModel
134:    // model: SubbasinTabModel

src/mmw/js/src/compare/views.js
345:    // model: ModificationsCollection
362:    // model: ScenarioModel

src/mmw/js/src/draw/views.js
161:    // model: ToolbarModel,
```